pkg/operation/botanist/monitoring.go

```diff
@@ -99,6 +99,9 @@ func (b *Botanist) DeploySeedMonitoring(ctx context.Context) error {
 		return err
 	}
 
+	// Need stable order before passing the dashboards to Grafana config to avoid unnecessary changes
+	kutil.ByName().Sort(existingConfigMaps)
```

Comment:
Awesome :-D

Reply:
The PR was a low-hanging fruit thanks to your groundwork :D